protos/in_toto_attestation/predicates/vsa/v0/vsa.proto

```diff
@@ -8,7 +8,7 @@ option go_package = "github.com/in-toto/attestation/go/predicates/vsa/v0";
 option java_package = "io.github.intoto.attestation.predicates.vsa.v0";
 
 
-// Proto representation of predicate type https://slsa.dev/verification_summary/v0.2
+// Proto representation of predicate type https://slsa.dev/verification_summary/v1
```

Comment:
I think this only changes the documented version, but what I think @trishankatdatadog is pointing out is that this whole VSA proto definition still follows the v0.2 spec. So I think we might need to create separate v1 proto.

Reply:
I will work to get a new v1 proto created

Reply:
created new v1